### ISSUE TITLE
10.3 breaks skip forward/backward

### ISSUE THREAD

**Issue by VoodooZ** [jellyfin/jellyfin#1267]
I've observed a similar problem back to 10.3 rc1
and sadly the newly released 10.3 final release does the same.
Skipping ahead (either by cursor or using my magic remote's cursor) causes the video to restart from the beginning.
Reverting to 10.2.2 fixes the problem.

Thanks in advance,

**Comment by bugfixin** [comment]
I'm not able to reproduce this in 10.3.0 - could you provide more information (reverse proxy, OS, client, logs, etc?)

**Comment by VoodooZ** [comment]
Sorry for the incomplete case description.
Will do tomorrow.

Thanks

**Comment by VoodooZ** [comment]
Tried it again on 10.3.1 and it still doesn't work.
OS is Centos 7.6.1810.
Running as portable binaries.
No reverse proxy.

My client of choice sadly still is Emby App under LG WebOS 3.
I've tried Screenplay and the result is the same.
But when I open the file directly via the native WebOS DLNA player it fast forwards without issue.

Although I did try a brand new Docker setup today for fun and it didn't change much.
Is it possible that my experimental h/w (nvidia) transcoding setup could affect that? I assume not for simple fast forwarding...

In each case, the video is played in DirectPlay mode.

Couldn't find anything much in logs but here's some:

```
[01:23:52] [INF] Setting range response values for /media/some_file.mkv. RangeRequest: bytes=0- Content-Range: bytes 0-3026671437/3026671438
[01:24:02] [WRN] HTTP Response 206 to 192.168.1.20. Time (slow): 0:00:18.7196863. http://192.168.1.2:8096/emby/Videos/94f05d357aac4600fb2c74f3ba6cc427/stream.mkv?Static=true&mediaSourceId=94f05d357aac4600fb2c74f3ba6cc427&deviceId=TW96aWxsYS81LjAgKFdlYjBTOyBMaW51eC9TbWFydFRWKSBBcHBsZVdlYktpdC81MzcuMzYgKEtIVE1MLCBsaWtlIEdlY2tvKSBDaHJvbWUvMzguMC4yMTI1LjEyMiBTYWZhcmkvNTM3LjM2IFdlYkFwcE1hbmFnZXJ8MTUzOTc0MzIyMTEzMQ11&Tag=f15f35f0dbd40591a6c2e4e62681df89
[01:24:11] [INF] User policy for stephane. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
```
I'll try again with debug logs to see if I can get anything more... But let me know if there's anything I can try in the meantime.

Thank you!

**Comment by cvium** [comment]
@VoodooZ is that the request _after_ skipping?

**Comment by VoodooZ** [comment]
It's pretty much the only thing that appeared relevant around the the time
I pressed the skip forward button yes.
Enabling the debug output didn't show much more strangely.

**Comment by anthonylavado** [comment]
This appears to be related to https://github.com/jellyfin/jellyfin-kodi/issues/21

The current theory is that adding the content-length header back will allow this to work as intended again.

**Comment by Matt07211** [comment]
I also experienced this problem on the android app, and what makes it worse is it spawns more ffmpeg process each time I try and seek

**Comment by anthonylavado** [comment]
To be clear, this is fixed in #1287, included in the next release.

**Comment by VoodooZ** [comment]
works fine again! thanks!
